Makefile

```diff
@@ -172,17 +172,18 @@ ALL_DOCS := $(shell find . -type f -name '*.md' -not -path './.github/*' -not -p
 markdown-link-check:
 	docker run --rm \
 		--mount 'type=bind,source=$(PWD),target=/home/repo' \
-		lycheeverse/lychee \
+		lycheeverse/lychee:sha-3a09227-alpine \
```

Comment:
This is not managed by renovate.

Something like a [`dependencies.Dockerfile`](https://github.com/open-telemetry/opentelemetry-go/blob/main/dependencies.Dockerfile) and [parsing](https://github.com/open-telemetry/opentelemetry-go/blob/2cce18995527b5bdfc8379497d3573259cb00a72/Makefile#L90-L91) should be considered.

Reply:
I believe it should. As this PR is proposed it is removing support for renovate to manage updates.

Reply:
Fixed in 4fe10486c62dce05bb622c9b4aac4c37b219f68f